lib/vsc/utils/mail.py

```diff
@@ -85,13 +103,20 @@ def _send(self,
         """
 
         try:
-            if self.mail_host:
-                logging.debug("Using %s as the mail host", self.mail_host)
-                s = smtplib.SMTP(self.mail_host)
+            if self.auth:
+                logging.debug("Using %s as the mail host with authenticatio", self.mail_host)
+                s = smtplib.SMTP(host=self.mail_host, port=self.mail_port)
```

Comment:
so this call can be put outside the if/else block when using default `''` and `0`